czkawka_core/src/duplicate.rs

```diff
@@ -292,16 +261,11 @@ impl DuplicateFinder {
                             };
 
                         // Checking expressions
-                        let mut found_expression: bool = false;
                         for expression in &self.excluded_items.items {
                             if Common::regex_check(expression, &current_file_name) {
-                                found_expression = true;
-                                break;
+                                break 'dir;
```

Comment:
This should be changed to `continue 'dir`
I know that `break` is in original code, but it is just a bug, because `'dir: for entry in read_dir ` iterates over all children and should be never break because it may skip some files/folders

Reply:
I went ahead and reverted the attempted fixes which mistakenly added the `break` instead of the original `continue`.